### ISSUE TITLE
LFG Schedule Command

### ISSUE THREAD

**Issue by wise-io** [wise-io/GolemHeart#18]
Add an /lfg or equivalent command. Functionality should be similar to [SpellBot](https://spellbot.io/), with the following additions:

- Add ability to schedule a game utilizing [Discord Scheduled Events](https://support.discord.com/hc/en-us/articles/4409494125719-Scheduled-Events).
- Add [epoch timestamp](https://pastebin.com/rJFE9yxq) to coordinate game among different time zones.
- Add link to spectate a game.
- Send reminders before event.

If a user uses the command without a schedule, functionality should be on par with SpellBot.

**Comment by wise-io** [comment]
Let's see if the SpellBot dev is interested in adding this functionality directly to SpellBot first: https://github.com/lexicalunit/spellbot/issues/683

**Comment by daLoach20** [comment]
Seems like discord.js is still working on implementing the needed functionality to allow this to happen: https://github.com/discordjs/discord.js/pull/6493

**Comment by wise-io** [comment]
> Seems like discord.js is still working on implementing the needed functionality to allow this to happen: https://github.com/discordjs/discord.js/pull/6493

Indeed. It's a very new Discord feature, so I'm waiting to see what will be possible with their implementation.
